src/build.sh

```diff
@@ -89,12 +89,18 @@ function kernel_build()
   fi
 
   if [ -x "$(command -v nproc)" ]; then
-    parallel_cores=$(nproc --all)
+    parallel_cores="-j$(nproc --all)"
   else
-    parallel_cores=$(grep -c ^processor /proc/cpuinfo)
+    parallel_cores="-j$(grep -c ^processor /proc/cpuinfo)"
   fi
 
-  command="make -j$parallel_cores ARCH=$arch $cross_compile"
+  if [[ -n "${options_values['OPTIMIZE']}" ]]; then
+    optimizations="CC=\"ccache gcc -fdiagnostics-color\" $parallel_cores"
```

Comment:
I would prefer that we handle `ccache` as a separate option. We can have an entry in the `etc/kworkflow_template.config` to enable it (e.g., `enable_ccache=no`). We can also have a new option under the `kw b` named `--ccache`. Finally, you need to update files in the folder `documentation/dependencies/` to make ccache explicit kw dependencies.